website/content/docs/sync/gcpsm.mdx

```diff
@@ -129,9 +129,23 @@ secrets.
 Regardless of the region limits on writes, synced secrets are always readable 
 globally when the client has the required permissions.
 
+### Encryption keys
+
+Vault can leverage customer-managed encryption keys (CMEK) to encrypt secrets in GCP Secret Manager instead of Google-managed keys
```

Comment:
suggestion to improve readability IIUC
```suggestion
Vault can leverage customer-managed encryption keys (CMEK) instead of Google-managed keys to encrypt secrets in GCP Secret Manager 
```